op-guide/tidb-v2.1-upgrade-guide.md

```diff
@@ -0,0 +1,117 @@
+---
+title: TiDB 2.1 Upgrade Guide
+summary: Learn how to upgrade from TiDB 2.0 (TiDB 2.0.1 or later) or TiDB 2.1 RC version to TiDB 2.1 GA version.
+category: deployment
+---
+
+# TiDB 2.1 Upgrade Guide
+
+This document describes how to upgrade from TiDB 2.0 (TiDB 2.0.1 or later) or TiDB 2.1 RC version to TiDB 2.1 GA version. TiDB 2.1 is not compatible with the Kafka version of TiDB-Binlog. If your current TiDB cluster has already been using the [Kafka version of TiDB-Binlog](../tools/tidb-binlog-kafka.md), you need to [upgrade it to the cluster version of TiDB-Binlog](../tools/tidb-binlog-cluster.md#tidb-binlog-upgrade).
+
+## Step 1: Install Ansible and dependencies on the Control Machine
```

Comment:
Is there a description of how to do a rolling update without Ansible?

Reply:
Can we describe in abstract terms what Ansible is doing?

Reply:
We do have such a description here:) https://github.com/pingcap/docs/blob/master/op-guide/ansible-deployment.md#overview